sbi/analysis/sensitivity_analysis.py

```diff
@@ -501,3 +502,53 @@ def project(self, theta: Tensor, num_dimensions: int) -> Tensor:
         projected_theta = torch.mm(theta, projection_mat)
 
         return projected_theta
+
+
+def posterior_shrinkage(prior_std, post_std):
+    """
+    Calculate the posterior shrinkage. The posterior shrinkage
+    quantifies how much the posterior distribution contracts
+    from the initial prior distribution.
+
+    Parameters
+    ----------
+    prior_std : float, array-like
```

Comment:
Typing should also be valid for tensors (i.e. vectors)

I guess one can define this metric also in a multivariate setting, just with the "total stddev" i.e. the sum.